raft.go

```diff
@@ -198,13 +267,8 @@ func NewRaft(conf *Config, fsm FSM, logs LogStore, stable StableStore, snaps Sna
 		}
 	}
 
-	// Construct the list of peers that excludes us
 	localAddr := trans.LocalAddr()
-	peers, err := peerStore.Peers()
-	if err != nil {
-		return nil, fmt.Errorf("failed to get list of peers: %v", err)
-	}
-	peers = ExcludePeer(peers, localAddr)
+	localGUID := localAddr // TODO: where should we get this from?
```

Comment:
My initial thought was that this would be provided as a new field in `conf`, and if that's empty then we'd use `localAddr` as a default and emit a deprecation warning that we'll stop defaulting it at some point in the future.